pkg/web_app/lib/src/api_client/api_client.dart

```diff
@@ -26,3 +27,17 @@ PubApiClient get unauthenticatedClient =>
 PubApiClient get client {
   return PubApiClient(_baseUrl, client: http.createClientWithCsrf());
 }
+
+Future<Map<String, dynamic>> requestJson(
```

Comment:
Probably best to document what this does in a  documentation comment.

And if we make `POST` the default verb, I'd suggest removing the `verb` option.
We could also make the `verb` a required property.

I'm just a bit confused, we'll think the method does something else in the future.

```suggestion
Future<Map<String, dynamic>> postJson(
```

Reply:
Updated this to `sendJson` + required method parameters, PTAL.